packaging.md

```diff
@@ -0,0 +1,155 @@
+# Packaging Applications for Deployment
+
+Once the application is built for production, it still needs to be packaged before it can be deployed to servers. There are several strategies for packaging Swift applications for deployment.
```

Comment:
```suggestion
Once an application is built for production, it still needs to be packaged before it can be deployed to servers. There are several strategies for packaging Swift applications for deployment.
```